c/azure-c-shared-utility

```diff
@@ -1 +1 @@
-Subproject commit 6eec79de491ea5fb3801255d27e54649ddbb2655
+Subproject commit 7daface8efc2d914538c1527fffb90894ff20b54
```

Comment:
First we need to have the PR for each of the subprojects.
To minimize iterations I think this option (build_tests) should appear as an option in the CMakeLists.txt, default should be on (I suspect that's what you did but I can't see those changes because they are in submodules.
If you add a --no-tests option in the Linux build script, we should have the same in the Windows one.